src/actions/notificationAction.js

```diff
@@ -1,41 +1,63 @@
-import { GET_PLATFORM_NOTIFICATIONS, GET_USER_NOTIFICATIONS } from './types'
-import axios from 'axios'
-import { errorHandler } from '../utils/errorHandler';
-import { setRequestStatus } from '../utils/setRequestStatus';
+import {
+  GET_PLATFORM_NOTIFICATIONS,
+  GET_USER_NOTIFICATIONS,
+  GET_PROPOSAL_NOTIFICATIONS,
+} from "./types";
+import axios from "axios";
+import { errorHandler } from "../utils/errorHandler";
+import { setRequestStatus } from "../utils/setRequestStatus";
 
 // GET NOTIFICATIONS FOR WHOLE PLATFORM AS WELL AS FOR A USER
 export const getAllNotifications = () => async (dispatch) => {
   try {
-    const res = await axios.get('/notification/org/all')
-    dispatch(setRequestStatus(false))
+    const res = await axios.get("/notification/org/all");
+    dispatch(setRequestStatus(false));
     if (res.status === 200) {
-      dispatch(setRequestStatus(true))
-      console.log('Whole platform notification ', res.data.notifications)
+      dispatch(setRequestStatus(true));
```

Comment:
are these formatting changes or code changes by linter did you make these?

Reply:
@vaibhavdaren These are the due to the extensions he is using or due to his vs code settings.
I have already mentioned that